docs/tutorials/optimizers_adamw.ipynb

```diff
@@ -0,0 +1,692 @@
+{
```

Comment:
<p>I worry  that saying "all the popular Deep Learning frameworks (Tensorflow, Pytorch) have implemented Adam with weight decay wrong." will become out of date (if it is not already). Could you remove that comment please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>